docs/deploy/Deploy-FhirConverterService.bicep

```diff
@@ -0,0 +1,193 @@
+@description('Location where the resources are deployed.')
+@allowed([
+  'australiaeast'
+  'canadacentral'
+  'centralindia'
+  'centralus'
+  'eastus'
+  'eastus2'
+  'francecentral'
+  'germanywestcentral'
+  'japaneast'
+  'koreacentral'
+  'northcentralus'
+  'northeurope'
+  'southcentralus'
+  'southeastasia'
+  'swedencentral'
+  'switzerlandnorth'
+  'uksouth'
+  'westeurope'
+  'westus2'
+  'westus3'
+])
+param location string
+
+@description('The name of the container app running the FHIR-Converter service.')
+param appName string
+
+@description('The name of the container apps environment where the app will run.')
+param envName string
+
+@description('Name of storage account containing custom templates. Leave blank if using default templates.')
+param templateStorageAccountName string = ''
+
+@description('Name of the container in the storage account containing custom templates. Leave blank if using default templates.')
+param templateStorageAccountContainerName string = ''
+
+@description('Minimum possible number of replicas per revision as the container app scales.')
+param minReplicas int = 0
+
+@description('Maximum possible number of replicas per revision as the container app scales.')
+param maxReplicas int = 30
+
+@description('CPU usage limit in cores.')
+param cpuLimit string = '1.0'
+
+@description('Memory usage limit in Gi.')
+param memoryLimit string = '2Gi'
+
+@description('If set to true, security will be enabled on the API endpoint.')
+param securityEnabled bool = false

```

Comment:
nit: can we rename to securityAuthenticationEnabled or just authenticationEnabled (if you choose this then maybe remove security prefix for the below ones too?). securityEnabled is vague imo.

Reply:
Okay nvm, I think this could also capture authorization in future. So we can keep it as is for now.